canvas/generate_one_case.py

```diff
@@ -0,0 +1,27 @@
+#   Copyright 2017 Google Inc. All Rights Reserved.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from __future__ import print_function
+import os
+import re
+import random
+import sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from grammar import Grammar
+
+my_grammar = Grammar()
+my_grammar.parse_from_file('canvas.txt')
+result_string = my_grammar.generate_root()
+
+print (result_string)
```

Comment:
Is this file really needed? Doesn't generator.py already have all that is done here?

Reply:
Well, that was quick silly test, instead of running every time generator.py with arguments and writing to files and stuff, I've used it to quickly check the output and modify the grammar based on what I needed!